app/lib/methods/subscriptions/rooms.js

```diff
@@ -301,7 +303,10 @@ export default function subscribeRooms() {
 					const roomState = store.getState().room;
 					// Delete and remove events come from this stream
 					// Here we identify which one was triggered
-					if (data.rid === roomState.rid && roomState.isDeleting) {
+					if (sub._raw.team_main && data.rid === roomState.rid && roomState.isDeleting) {
+						store.dispatch(removedRoom());
+						EventEmitter.emit(LISTENER, { message: I18n.t('Left_The_Team_Successfully') });
```

Comment:
Can you raise this alert here instead? https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/sagas/room.js#L54